src/components/KTabs/KTabs.vue

```diff
@@ -44,10 +44,7 @@
 <script lang="ts">
 import { defineComponent, ref, watch, PropType } from 'vue'
 
-export interface Tab {
-  hash: string
-  title: string
-}
+import type { Tab } from '@/types/tabs'
```

Comment:
nit: import from the root
```suggestion
import type { Tab } from '@/types'
```

Reply:
Changed in https://github.com/Kong/kongponents/pull/1250/commits/3d667c17cf9cc0acdeaf7b4a56fa35cef093a177.